shunit2

```diff
@@ -1140,11 +1140,23 @@ oneTimeSetUp
 command [ $? -eq ${SHUNIT_TRUE} ] \
     || _shunit_fatal "oneTimeSetUp() returned non-zero return code."
 
-# Execute the suite function defined in the parent test script.
-# DEPRECATED as of 2.1.0.
-suite
+# Command line selected tests or suite selected tests
+if command [ "$#" -ge 2 ]; then
+  # arguemnt $1 is either the filename of tests or '--'; either way, skip it.
+  shift
+  # arguments $2 .. $# are assumed to be test function names
+  # interate through all remaining args in "$@" in a POSIX (likely portable) way
+  # helpful tip: https://unix.stackexchange.com/questions/314032/how-to-use-arguments-like-1-2-in-a-for-loop
+  for arg_test_ do
```

Comment:
The arg_test_ variable doesn't fit the variable name standard for shunit2.
https://github.com/kward/shunit2/wiki/Coding-standards

```suggestion
  for shunit_arg_ do
```

Reply:
Updated to follow coding conventions; see commit @6f8bc7a.